independent-publisher-connectors/AADInvitationManager/apiProperties.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "https://graph.microsoft.com"
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://graph.microsoft.com"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
```

Comment:
@Laskewitz Did you read the checklist mentioned in the description of your PR ?

![image](https://user-images.githubusercontent.com/56276394/125004114-aa392300-e00d-11eb-9a0b-a2ac9c7d4deb.png)

You can provide `"#da3b01"` which is default iconBrandColor of Individual Publisher connector.

Reply:
Fixed in https://github.com/microsoft/PowerPlatformConnectors/pull/924/commits/a70220abaa9f57a919431035fd626b6d97d91e25